inc/tinyMce.php

```diff
@@ -84,15 +84,35 @@ function getConfig()
                 'title' => 'Buttons',
                 'icon' => '',
                 'items' => [
+                    [
+                        'title' => 'Button',
+                        'classes' => 'button',
+                        'selector' => 'a,button'
```

Comment:
Perhaps the selectors should have a white space after the comma? -> `a, button`

Reply:
Good point, though I followed the TinyMCE way without separator and thus would keep it like that: https://www.tiny.cloud/docs/configure/content-formatting/#example